snapshot/modify_space.js

```diff
@@ -9,76 +9,54 @@
 
 const { ethers } = require('ethers');
 const snapshot = require('@snapshot-labs/snapshot.js');
+const dotenv = require('dotenv');
+
+dotenv.config();
 
 const ethPrivateKey = process.env.ETH_PRIVATE_KEY;
-const primaryRpc = process.env.PRIMARY_RPC;
-const secondaryRpc = process.env.SECONDARY_RPC;
+const primaryRpc = process.env.PRIMARY_RPC_URL;
+const secondaryRpc = process.env.SECONDARY_RPC_URL;
 
-const maxRetries = 3; // Number of retry attempts
-const initialRetryDelay = 5000; // Initial delay between retries in milliseconds (e.g., 5000ms = 5s)
+const hub = process.env.SNAPSHOT_HUB;
+const snapshotSpace = process.env.SNAPSHOT_SPACE;
+const network = process.env.NETWORK_ID || '11155111'; // Fallback network ID
```

Comment:
is this the only fallback value that needs to exist for this script to work?

Reply:
> is this the only fallback value that needs to exist for this script to work?

That, and the RPC URLs, outside of those no I don't think so.